orchestrator-salt/src/main/resources/salt/salt/cloudera/agent/init.sls

```diff
@@ -47,6 +48,19 @@ setup_autotls_token:
     - after: "use_tls=.*"
     - backup: False
 
+{% if internal_loadbalancer_san|length > 0 %}
+
+setup_autotls_san:
+  file.line:
+    - name: /etc/cloudera-scm-agent/config.ini
```

Comment:
What is this change doing?

Reply:
The auto-tls certificates only have the hostname before this change. IF we are calling the Oozie API using load balancer name, it will fail for the hostname verification. So the load-balancer is added as a SAN to make the TLS certificate verification pass.